### ISSUE TITLE
So many overrides

### ISSUE THREAD

**Issue by wirthual** [bundesAPI/apis#23]
Does anyone have an overview why we have so many overrides?

**Comment by AndreasFischer1985** [comment]
That's because our API-documentations don't specify the (custom) parameter x-office yet and consequently [refresh.js](https://github.com/bundesAPI/apis/blob/main/refresh.js) is not able to extract information on the office correctly.

**Comment by t-huyeng** [comment]
Would it not be enough to only specify the x-office parameter and the github url instead of all the information in the `overrides.json`?

I think this would also clarify what is missing and what is added.

**Comment by wirthual** [comment]
Yes, that's correct. If the entries of the `overrides.json` would get merged into entries extracted from the `openapi.yaml`  that would be a good solution. At the moment, the full entry is simply replaced.

Ultimately we should just x-office directly to the openapi.yaml to overcome the entry in the overrides.json.

**Comment by t-huyeng** [comment]
I think they are getting merged into the exiting entries. 
I tried a simple example here : https://replit.com/@tiib/override-bundesapis#index.js

So we could just remove other parameter from the `overrides.json` and it should still work.

**Comment by wirthual** [comment]
Cool. Then it seems we can reduce it to only the needed keys.

 However some do not have a description in their openapi.yaml so we need to be careful which entries to shorten. E.g. die destatis-api has no description in the `openapi.yaml`

**Comment by t-huyeng** [comment]
As described in #30 we could get rid of at least a few overrides if we could change the description and webpage of the repositories.
